src/app/(profile)/profile.mappers.ts

```diff
@@ -172,3 +172,14 @@ export const mapSpecializations = (
     value: specialization,
   }))
 }
+
+//HOURLY RATE
+export const hourlyRateOptions: DropdownOption[] = [
```

Comment:
**nit:** This is not a mapper, let's make a mapping function of it or let's move it closer to the component which uses it